### PR TITLE
support for ARM64 agent & azure project

### PR DESCRIPTION
**Pull request by shaharhayon**

### CODE REVIEW

File: agent-setup.sh [hunk 1]
```diff
@@ -44,7 +46,7 @@ export AGENT_INSTALL_DIR=$AGENT_USER_HOME/agent
 
 echo "Downloading agent"
 mkdir -p $AGENT_INSTALL_DIR
-curl -f -o $AGENT_INSTALL_DIR/agent.tar.gz https://vstsagentpackage.azureedge.net/agent/$AGENT_VERSION/vsts-agent-linux-x64-$AGENT_VERSION.tar.gz
+curl -f -v -o $AGENT_INSTALL_DIR/agent.tar.gz https://vstsagentpackage.azureedge.net/agent/$AGENT_VERSION/vsts-agent-linux-$ARCH-$AGENT_VERSION.tar.gz
```
Comment (EugenMayer) [inline agent-setup.sh]: Please remove the verbose flag

**Review by EugenMayer — COMMENTED** [review]
Thank you for the contribution! One tiny bit and it is good to go!

File: agent-setup.sh (hunk 1 above)
Comment (shaharhayon) [reply agent-setup.sh]: fixed, debugging residue 🥇

**Comment by EugenMayer** [comment]
Thank you for the contribution!

**Pull request merged**